gui/src/components/health/cortx-health-graphical.vue

```diff
@@ -0,0 +1,544 @@
+              break;
+            case "unknown" :
+              imageLink = require("@/assets/grey-icon.svg/");
+              break;
+          }
+          return imageLink;
+        })
+        .attr("x", "10px")
+        .attr("y", "55px")
+        .attr("width", "30px")
+        .attr("height", "30px");
+      nodeEnter
+        .append("text")
+        .attr("id", function(d: any) {
+          return "nodetextId" + d.id + d.depth;
+        })
+        .attr("dy", "70px")
+        .attr("x", function(d: any) {
+          return 50;
+        })
+        .style("font-size", "14px")
+        .text(function(d: any) {
+          return d.data.resource;
+        })
+        .style("font-weight", "bold")
+        .style("text-transform", "capitalize");
+      nodeEnter
+        .append("text")
+        .attr("id", function(d: any) {
+          return "nodetextId1" + d.id + d.depth;
+        })
+        .attr("dy", "90px")
+        .attr("x", function(d: any) {
+          return 50;
+        })
+        .style("font-size", "12px")
+        .text(function(d: any) {
+          if (d.data.id.length > 5) {
+            return d.data.id.substring(0, 5) + "...";
+          } else {
+            return d.data.id;
+          }
+        });
+      nodeEnter
+        .append("circle")
+        .attr("class", "node")
+        .attr("r", function(d: any) {
+          if (d.data.children && d.data.children.length) {
+            return "10";
+          } else {
+            return "0";
+          }
+        })
+        .attr("cx", 155)
+        .attr("cy", 40)
+        .style("fill", "none")
+        .style("stroke", "#6EBE49")
+        .style("stroke-width", 1)
+        .style("cursor", "pointer")
+        .on("click", click);
+      nodeEnter
+        .append("text")
+        .attr("dy", "45px")
+        .attr("x", function(d: any) {
+          if (
+            d.data.children &&
+            d.data.children.length &&
+            d.children &&
+            d.children.length
+          ) {
+            return "152.5";
+          } else {
+            return "150";
+          }
+        })
+        .text(function(d: any) {
+          if (d.children) {
+            return "-";
+          } else if (!d.children && d.data.children && d.data.children.length) {
+            return "+";
+          } else {
+            return " ";
+          }
+        })
+        .style("cursor", "pointer")
+        .on("click", click);
+      // UPDATE
+      let nodeUpdate = nodeEnter.merge(node);
+      // Transition to the proper position for the node
+      nodeUpdate
+        .transition()
+        .duration(duration)
+        .attr("transform", function(d: any) {
+          return "translate(" + d.y + "," + d.x + ")";
+        });
+      // Remove any exiting nodes
+      let nodeExit = node
+        .exit()
+        .transition()
+        .duration(duration)
+        .attr("transform", function(d: any) {
+          return "translate(" + source.y + "," + source.x + ")";
```

Comment:
Please use template literal.

Reply:
Updated to template literal everywhere in a code.